.github/workflows/backend-integration-test.yml

```diff
@@ -9,8 +9,8 @@ on:
     branches: [ "main", "branch-*" ]
 
 concurrency:
-  group: ${{ github.worklfow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: ${{ github.event_name == 'pull_requests' }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
What is the purpose of this change?

Reply:
1. typo: worklfow 
2. There is not event name 'pull_requests', only `pull_request`
3. `cancel-in-progress` means under which event will we cancel pipelines in progress,  there are `push`, `pull_request` and other, `true` means any changes to the PR branch will cancel in process pipelines